get-started.md

```diff
@@ -14,8 +16,14 @@ We offer two convenient methods to incorporate Mapzen basemaps into your project
 For example:
 
 ```
+L.Mapzen.apiKey = 'mapzen-xxxxxx';
+
 var map = L.Mapzen.map('map', {
-  scene: L.Mapzen.BasemapStyles.Refill
+  tangramOptions: {
+    scene: {
+      import: L.Mapzen.BasemapStyles.Refill,
```

Comment:
In the API page in this repo we follow this import line with:
`      global: { sdk_mapzen_api_key: 'mapzen-xxxxxx' }`

It's not clear to me if that is necessary / matches the recommended Mapzen.js design pattern (I've had mixed results).